src/common/main.lua

```diff
@@ -324,6 +324,34 @@ function ant.init()
 			Names = msg.Tags.Names,
 		})
 	end)
+
+	Handlers.prepend("_bootAnt", function(msg)
+		return msg.Tags.Type == "Process" and Owner == msg.From
+	end, function(msg)
+		if msg.Data and msg.Tags["Initialize-State"] then
```

Comment:
Rather than depending on the `"Initialize-State"` flag (which is in this case is actually not checking its value adequately), we'll instead examine/validate the input msg.Data and if it looks correct for boot loading, then proceed and use it.

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/57/commits/7f04c36b536011cec5286215a0ef1bd830c54ee2